wolftpm/tpm2_wrap.h

```diff
@@ -1308,6 +1343,10 @@ WOLFTPM_API int wolfTPM2_VerifyHash_ex(WOLFTPM2_DEV* dev, WOLFTPM2_KEY* key,
     const byte* sig, int sigSz, const byte* digest, int digestSz,
     int hashAlg);
 
+WOLFTPM_API int wolfTPM2_VerifyHash_ex2(WOLFTPM2_DEV* dev, WOLFTPM2_KEY* key,
```

Comment:
Instead of ex2 can this be wolfTPM2_VerifyHashTicket or something more descriptive?